.prettierignore

```diff
@@ -0,0 +1,5 @@
+*.md
+azure-*.yml
+docs/source/_templates/*
+homeassistant/components/*/.translations/*.json
```

Comment:
These are auto generated by a bot. They should never be manually edited and so don't need to be linted.

Reply:
nevermind, these are ignored.